File: exercises/javascript/vehicle.js
let Vehicle = class Vehicle {
  constructor(co2Emissions, fuelType, dateOfFirstRegistration, listPrice) {
    this.co2Emissions = co2Emissions;
    this.fuelType = fuelType;
    this.dateOfFirstRegistration = dateOfFirstRegistration;
    this.listPrice = listPrice;
  }

}

module.exports = { Vehicle: Vehicle }

File: exercises/dotnet/TaxCalculator/Vehicle.cs
using System;

namespace TaxCalculator
{
    public class Vehicle
    {
        public int Co2Emissions { get; }
        public FuelType FuelType { get; }
        public DateTime DateOfFirstRegistration { get; }
        public int ListPrice { get; }

        public Vehicle(int co2Emissions, FuelType fuelType, DateTime dateOfFirstRegistration, int listPrice)
        {
            Co2Emissions = co2Emissions;
            FuelType = fuelType;
            DateOfFirstRegistration = dateOfFirstRegistration;
            ListPrice = listPrice;
        }
    }
}
